cli/tests/test_cmd_setup_redaction.py

```
@@ -0,0 +1,328 @@
+        warnings=(),
+        locked_profiles=(("managed-enterprise-ai-defense", ("sensitive",)),),
+        mutation_paths=("$.observability.defaults.redaction_profile",),
+    )
+    calls = []
+    monkeypatch.setattr(
+        cmd_setup_redaction,
+        "preview_redaction_mutations",
+        lambda *_args, **_kwargs: preview,
+    )
+
+    def mutate(path, mutations, **kwargs):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `mutate`

(ANN202)